packages/components/src/navigator/test/index.tsx

```diff
@@ -19,14 +19,13 @@ import {
 	NavigatorScreen,
 	NavigatorButton,
 	NavigatorBackButton,
-	NavigatorToParentButton,
 	useNavigator,
 } from '..';
 import type { NavigateOptions } from '../types';
 
 const INVALID_HTML_ATTRIBUTE = {
-	raw: ' "\'><=invalid_path',
-	escaped: " &quot;'&gt;<=invalid_path",
+	raw: '/ "\'><=invalid_path',
+	escaped: "/ &quot;'&gt;<=invalid_path",
```

Comment:
Because of "going back" always behaving like "go to parent", all paths need to start with `/` in order to work well with the component's [pattern matching logic](https://github.com/WordPress/gutenberg/blob/378290a8b7cacdbdaa441c3c3f43436464f02dba/packages/components/src/navigator/utils/router.ts)